cli/src/main/kotlin/ls/kafka/backup/s3/BackupBucket.kt

```diff
@@ -136,41 +87,39 @@ class BackupBucket(val bucket: String, s3Config: S3Config, val kafkaConfig: Prop
         }
     }
 
-    suspend fun getRecords(
+    private suspend fun getRecords(
         topicPattern: String,
         partition: Int? = null,
-    ): List<Record> = find(
+    ): List<BackupRecord> = find(
         topicPattern,
         partition = partition,
     ).map { backupFile(it) }.flatMap { backupFile ->
-        backupFile.records().map { it.withTopic(backupFile.topic) }
+        backupFile.records().map { BackupRecord(backupFile.topic, it) }
     }
 
-
-    suspend fun validate() {
+    private suspend fun validate() {
         s3.headBucket {
             bucket = this@BackupBucket.bucket
         }
     }
 
-    @Suppress("ForbiddenMethodCall")
```

Comment:
I think removing this causes detekt to fail, because of the `println`. I initially used println instead of a logger because I dont think we need the whole ts + package name from a logger in a cli tool. wdyt?

Reply:
yes you are right. I removed this rule in Detekt instead.